Segments/Blackburrow.mapproj

```diff
@@ -122206,6 +122202,208 @@ if (spokeRecently(source))
 
     source.Say($"What are you doing down here?! I will silence you before you can tell your kind what is happening here!"); 
 
+]]></block>
+        <block><![CDATA[]]></block>
+      </script>
+    </entity>
+    <entity name="Level12GnollExcavator">
+      <script name="OnSpawn" enabled="true">
+        <block><![CDATA[]]></block>
+        <block><![CDATA[
+	var sooth = new Troll()
+    {
+        Name = "Gnoll Excavator",
+        MaxHealth = 120, Health = 120,
+        MaxMana = 12, Mana = 12,
+        BaseDodge = 21,
+        Body = 390,
+        BasePenetration = ShieldPenetration.Light,
+        Experience = 7000,
+        VisibilityDistance = 3,
+        LightningResistance = 20,
+        HideDetection = 32
+    };
+    
+    
+    sooth.AddStatus(new NightVisionStatus(sooth));
+        
+    sooth.Attacks = new CreatureAttackCollection()
+    {
+        new CreatureBasicAttack(12, 15, 20)
+    };
+    
+    sooth.Spells = new CreatureSpellCollection()
+    {
+        new CreatureSpell<DeathSpell>(
+            skillLevel: 6, cost: 6, 
+            mantra: SpellHelper.GenerateMantra(), instantCast: false),
+        new CreatureSpell<ConcussionSpell>(
```

Comment:
Concern

Reply:
Once it goes in I'll do a run through. Still got to rename more entities and refine some damage on some mobs.